app/schemas/org.simple.clinic.AppDatabase/118.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
```

Comment:
Just a suggestion for future, don't have to change now. "Update json file" seems way to vague, commit message should mention "Update database schema JSON".

Reply:
sure. will keep in mind.